src/components/_global/BalTable/BalTable.vue

```diff
@@ -343,27 +344,32 @@ watch(
         <!-- end pinned rows -->
 
         <!-- begin data rows -->
-        <BalTableRow
+        <template
           v-for="(dataItem, index) in unpinnedData"
           :key="`tableRow-${dataItem.id ?? index}`"
-          :class="
-            props.getTableRowClass
-              ? props.getTableRowClass(dataItem, index)
-              : undefined
-          "
-          :data="dataItem"
-          :columns="filteredColumns"
-          :onRowClick="onRowClick"
-          :link="link"
-          :href="href"
-          :sticky="sticky"
-          :isColumnStuck="isColumnStuck"
         >
-          <slot :name="'named'" v-bind="dataItem"></slot>
-          <template v-for="(_, name) in $slots" #[name]>
-            <slot :name="name" v-bind="dataItem" />
-          </template>
-        </BalTableRow>
+          <BalTableRow
+            v-if="!showingRowsIdx || index <= showingRowsIdx"
```

Comment:
This may cause problems if the showingRowsIdx === 0? !0 == true

Reply:
actually this is the point. if showingRowsIdx is 0 or undefined we want rows to be shown

Reply:
also showingRowsIdx === 0 doesn't make much sense